core/evalir.ml

```diff
@@ -277,7 +277,7 @@ struct
            with
                  UnknownProcessID _ ->
                    (* FIXME: printing out the message might be more useful. *)
-                   internal_error("Couldn't deliver message because destination process has no mailbox.")) >>= fun _ ->
+                   raise (internal_error("Couldn't deliver message because destination process has no mailbox."))) >>= fun _ ->
```

Comment:
Would it be easy to do what the FIXME is saying?

Reply:
Yep. Done so, thanks.